arch/loongarch/loongarch_features.c

```diff
@@ -0,0 +1,27 @@
+/* loongarch_features.c -- check for LoongArch features.
+ *
+ * Copyright (C) 2025 Vladislav Shchapov <vladislav@shchapov.ru>
+ *
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#include "zbuild.h"
+#include "loongarch_features.h"
+
+#include <larchintrin.h>
+
+/*
+ * https://loongson.github.io/LoongArch-Documentation/LoongArch-Vol1-EN.html
+ *
+ * Word number Bit number  Annotation  Implication
+ * 0x1         25          CRC         1 indicates that support CRC instruction
+ * 0x1         6           LSX         1 indicates support for 128-bit vector extension
+ * 0x1         7           LASX        1 indicates support for 256-bit vector expansion
```

Comment:
All rows in `Implication` column should start with just `1 = supports` or `1 indicates support for`... There is no reason for the first row to use different text.